schema-registry/core/src/main/java/com/hortonworks/registries/schemaregistry/SchemaMetadataStorable.java

```diff
@@ -80,6 +81,12 @@
      */
     private SchemaCompatibility compatibility = SchemaCompatibility.DEFAULT_COMPATIBILITY;
 
+
+    /**
+     * Validation level of this schema instance
+     */
+    private SchemaValidationLevel validation = SchemaValidationLevel.DEFAULT_VALIDATION_LEVEL;
```

Comment:
Can it be renamed to `validationLevel` and respective get/set and schema field name etc?